src/mergeContact.spec.js

```diff
@@ -0,0 +1,445 @@
+            familyName: 'Doe'
+          }
+        ]
+      }
+      const result = mergeContact(cozyContact, googleContact, {
+        preferGoogle: true
+      })
+      expect(result.name.givenName).toEqual('Peter')
+      expect(result.name.familyName).toEqual('Doe')
+    })
+
+    it('should add simple field', () => {
+      const cozyContact = {}
+      const googleContact = {
+        organizations: [
+          {
+            name: 'Cozy cloud'
+          }
+        ]
+      }
+      const result = mergeContact(cozyContact, googleContact, {
+        preferGoogle: true
+      })
+      expect(result.company).toEqual('Cozy cloud')
+    })
+
+    it('should add a value and a label in a multiple field', () => {
+      const cozyContact = {
+        email: [
+          {
+            address: 'john.doe@posteo.net',
+            label: 'Personal',
+            type: 'personal',
+            primary: false
+          }
+        ]
+      }
+      const googleContact = {
+        emailAddresses: [
+          {
+            value: 'john.doe@posteo.net',
+            formattedType: 'Personal',
+            type: 'personal',
+            metadata: {
+              primary: false
+            }
+          },
+          {
+            value: 'john.doe@cozycloud.cc',
+            formattedType: 'Professional',
+            type: 'professional',
+            metadata: {
+              primary: true
+            }
+          }
+        ]
+      }
+      const result = mergeContact(cozyContact, googleContact, {
+        preferGoogle: true
+      })
+      expect(result.email).toEqual([
+        {
+          address: 'john.doe@posteo.net',
+          label: 'Personal',
+          type: 'personal',
+          primary: false
+        },
+        {
+          address: 'john.doe@cozycloud.cc',
+          label: 'Professional',
+          type: 'professional',
+          primary: true
+        }
+      ])
+    })
+
+    it('should remove a value in a multiple field', () => {
+      const cozyContact = {
+        email: [
+          {
+            address: 'john.doe@posteo.net',
+            label: 'Personal',
+            type: 'personal',
+            primary: false
+          },
+          {
+            address: 'john.doe@cozycloud.cc',
+            label: 'Professional',
+            type: 'professional',
+            primary: true
+          }
+        ]
+      }
+      const googleContact = {
+        emailAddresses: [
+          {
+            value: 'john.doe@posteo.net',
+            formattedType: 'Personal',
+            type: 'personal',
+            metadata: {
+              primary: true
+            }
+          }
+        ]
+      }
+      const result = mergeContact(cozyContact, googleContact, {
+        preferGoogle: true
+      })
+      expect(result.email).toEqual([
+        {
+          address: 'john.doe@posteo.net',
+          label: 'Personal',
+          type: 'personal',
+          primary: true
+        }
```

Comment:
Since we're always using the same object, can we create some kind of global object somewhere?